docs/main.mjs

```diff
@@ -23,3 +29,37 @@ document.querySelectorAll('.copy-icon').forEach(icon => {
         }, 750)
     })
 })
+
+backToTopBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    window.scrollTo(0, 0);
+})
+
+darkCheck.addEventListener('click', () => {
+    if(darkCheck.checked){
+        document.body.className = 'dark';
+        localStorage.setItem('animate-css', 'dark');
+        moon.style.fill = '#e0e0e0';
+        sun.style.fill = '#e0e0e0';
+    }else{
+        document.body.className = '';
+        localStorage.removeItem('animate-css');
+        moon.style.fill = '#111';
+        sun.style.fill = '#111';
+    }
+})
+
+window.addEventListener("scroll", () => {
```

Comment:
please add `{ passive: true }` as a third argument to the `addEventListener` function. [See more about it here](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#improving_scrolling_performance_with_passive_listeners).